src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,390 @@
+| B            |  1.1.1.2     | 4500      | D              | 2.1.1.1        | 4502       | yes   |
+
+The recommended configuration for the gateways behind the on-premises router which has
+a single external IP with no IP routing or mapping to the private network is to have a
+dedicated and distinct port number for the NATT discovery protocol (as well as the encapsulation)
+
+```bash
+kubectl annotate node $GWC --kubeconfig C gateway.submariner.io/natt-discovery-port=4491
+kubectl annotate node $GWC --kubeconfig C gateway.submariner.io/udp-port=4501
+kubectl annotate node $GWD --kubeconfig D gateway.submariner.io/natt-discovery-port=4492
+kubectl annotate node $GWD --kubeconfig D gateway.submariner.io/udp-port=4502
+
+# restart the gateways to pick up the new setting
+for cluster in C D;
+do
+  kubectl delete pod -n submariner-operator -l app=submariner-gateway --kubeconfig $cluster
+done
+```
+
+{{% notice warning %}}
+If **HA** is configured on the on-premise clusters, **each gateway** behind the 2.1.1.1 router
+**should have a dedicated UDP port**. For example if we had 2 clusters and 2 gateways on each cluster, 4
+ports would be necessary.
+{{% /notice %}}
+
+#### Router Port Mapping
+
+Under this configuration it's important to map the UDP ports on the 2.1.1.1 router to the private
+IPs of the gateways.
+
+| External IP | Port | Internal IP   | Port | Protocol |
+|:------------|:-----|:--------------|:-----|:---------|
+| 2.1.1.1     | 4501 | 192.168.0.4   | 4501 | UDP      |
+| 2.1.1.1     | 4491 | 192.168.0.4   | 4491 | UDP      |
+| 2.1.1.1     | 4502 | 192.168.128.4 | 4502 | UDP      |
+| 2.1.1.1     | 4492 | 192.168.128.4 | 4492 | UDP      |
+
+{{% notice note %}}
+Without port mapping it's entirely possible that the connectivity will be established without
+issues. This can happen because the router's NAT will not generally modify the source port
+of the outgoing UDP packets, and future packets arriving on this port will be redirected to the
+internal IP which initiated connectivity. However if the 2.1.1.1 router randomizes the source port on
+NAT or if other applications on the internal network were already using the 4501-4502 or 4491-4492
+ports, the remote ends would not be able to contact gateway C or D over the expected ports.
+{{% /notice %}}
+
+#### Alternative to Port Mapping
+
+If port mapping is not possible, we can enable a server/client model for connections
+where we designate the clusters with a dedicated public IP or the clusters with the
+ability to get mapped ports as **preferred servers**. In this way, only the non-preferred
+server clusters will initiate connections to the preferred server clusters.
+
+For example, given clusters A, B, C, and D, we designate A and B as preferred servers:
+
+```bash
+subctl join --kubeconfig A --preferred-server .... broker_info.subm
+subctl join --kubeconfig B --preferred-server .... broker_info.subm
+``
```

Comment:
```suggestion
```bash
subctl join --kubeconfig A --preferred-server .... broker_info.subm
subctl join --kubeconfig B --preferred-server .... broker_info.subm
```
```

Reply:
There is a missing ` at the end of the bash section